easybuild/easyconfigs/c/Chemaxon-Marvin/Chemaxon-Marvin-21.14.eb

```diff
@@ -30,11 +30,11 @@ docurls = [
 
 toolchain = SYSTEM
 
-# Download and license at https://chemaxon.com/products/marvin
-# Once logged in download files can be found here:
-# https://chemaxon.com/products/marvin/download#download
 sources = ['marvin_linux_%(version)s.rpm']
 checksums = ['3102f30479a365758fd14ca5efcdacfe0624db80a35b146ba9a5d07e948bd8dc']
+download_instructions = f"""{name} requires license and manual download from {homepage}
+Once logged in download files can be found at https://chemaxon.com/products/marvin/download#download
```

Comment:
I see the homepage removed the "/products/" section, I wonder if this link is correct, i suspect not. I didn't wanna register to check, but the homepage looks very straight forward to use, so I don't think we need to specify such a link at all. 
```suggestion
```

Reply:
3d59794